mantle/kola/tests/misc/network.go

```diff
@@ -299,79 +320,115 @@ var (
+		if [[ -z $connections_folder ]]; then
+		  connections_folder=/etc/NetworkManager/system-connections
+		fi
+		rm -rf ${connections_folder}/*
+
+		if [[ ! -f /boot/mac_addresses ]] ; then
+		  echo "no mac address configuration file found .. exiting"
+		  exit 0
+		fi
+
+		if [[ -d /var/ovsbond ]]; then
+		  echo "Loading OVS old profile"
+		  cp -r /var/ovsbond/* ${connections_folder}
+		  systemctl restart NetworkManager
+		fi
+
+		if [[ $(nmcli conn | grep -c ovs) -eq 0 ]]; then
+		  echo "configure ovs bonding"
+		  ovs-vsctl --if-exists del-br brcnv
+		  primary_mac=$(cat /boot/mac_addresses | awk -F= '/PRIMARY_MAC/ {print $2}')
+		  secondary_mac=$(cat /boot/mac_addresses | awk -F= '/SECONDARY_MAC/ {print $2}')
+
+		  default_device=""
+		  secondary_device=""
+		  profile_name=""
+		  secondary_profile_name=""
+		  for dev in $(nmcli device status | awk '/ethernet/ {print $1}'); do
+			dev_mac=$(nmcli -g GENERAL.HWADDR dev show $dev | sed -e 's/\\//g' | tr '[A-Z]' '[a-z]')
+			case $dev_mac in
+			  $primary_mac)
+				default_device=$dev
+				profile_name=$(nmcli -g GENERAL.CONNECTION dev show $dev)
+				;;
+			  $secondary_mac)
+				secondary_device=$dev
+				secondary_profile_name=$(nmcli -g GENERAL.CONNECTION dev show $dev)
+				;;
+			  *)
+				;;
+			 esac
+		  done
+		  echo -e "default dev: $default_device ($profile_name)\nsecondary dev: $secondary_device ($secondary_profile_name)"
+
+		  mac=$(sudo nmcli -g GENERAL.HWADDR dev show $default_device | sed -e 's/\\//g')
+
+		  ovs-vsctl --if-exists del-br brcnv
+		  # code.engineering
+		  nmcli conn add type ovs-bridge conn.interface brcnv
+		  nmcli conn add type ovs-port conn.interface brcnv-port master brcnv
+		  nmcli conn add type ovs-interface conn.id brcnv-iface conn.interface brcnv master brcnv-port ipv4.method auto ipv4.dhcp-client-id "mac" connection.autoconnect no 802-3-ethernet.cloned-mac-address $mac
+
+		  # make bond
+		  nmcli conn add type ovs-port conn.interface bond0 master brcnv ovs-port.bond-mode balance-slb
+		  nmcli conn add type ethernet conn.interface $default_device master bond0 con-name ovs-slave-1
+		  nmcli conn add type ethernet conn.interface $secondary_device master bond0 con-name ovs-slave-2
+		  nmcli conn down "$profile_name" || true
+		  nmcli conn mod "$profile_name" connection.autoconnect no || true
+		  nmcli conn down "$secondary_profile_name" || true
+		  nmcli conn mod "$secondary_profile_name" connection.autoconnect no || true
+		  if ! nmcli conn up brcnv-iface; then
+			  nmcli conn up "$profile_name" || true
+			  nmcli conn mod "$profile_name" connection.autoconnect yes
+			  nmcli conn up "$secondary_profile_name" || true
+			  nmcli conn mod "$secondary_profile_name" connection.autoconnect yes
+			  nmcli c delete $(nmcli c show |grep ovs-cnv |awk '{print $1}') || true
+		  else
+			  nmcli conn mod brcnv-iface connection.autoconnect yes
+			  nmcli conn up ovs-slave-2
+			  rm -r /var/ovsbond || true
```

Comment:
@RamLavi Dont you want to change the command to  "rm -rf /var/ovsbond || true" to avoid errors?

Reply:
Totally

Reply:
Your choice :)

Reply:
let's keep the `|| true` then.

Reply:
Cool, I think you can close this review